adapter/internal/oasparser/model/open_api.go

```diff
@@ -125,12 +125,16 @@ func setResourcesOpenAPI(openAPI openapi3.Swagger) ([]Resource, error) {
 	val, found := resolveAPILevelDisableSecurity(openAPI.ExtensionProps)
 	if openAPI.Paths != nil {
 		for path, pathItem := range openAPI.Paths {
+			// Checks for resource level security. (security is disabled in resource level using x-wso2-disable-security extension)
+			isResourceLvlSecurityDisabled, foundInResourceLevel := resolveAPILevelDisableSecurity(pathItem.ExtensionProps)
```

Comment:
shall we rename resolveAPILevelDisableSecurity method as it is not only used to resolve apilevel